res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1572,60 +1810,25 @@ PioneerDDJSX.playLed = function(value, group, control) {
     PioneerDDJSX.nonPadLedControl(group, PioneerDDJSX.nonPadLeds.shiftPlay, value);
 };
 
-PioneerDDJSX.playFromCuePoint = false;
-
-PioneerDDJSX.playIllumination = function(value, group, control) {
+PioneerDDJSX.wheelLeds = function(value, group, control) {
     var deck = PioneerDDJSX.channelGroups[group],
-        cue = PioneerDDJSX.cuePressed[deck],
-        trackLoaded = 0;
+        duration = engine.getValue(group, "duration"),
+        remainingTime = duration - (value * duration),
+        speed = 0x28,
+        wheelPos = 0x01;
 
-    trackLoaded = engine.getValue(group, "track_loaded");
-    if (!trackLoaded) {
-        return;
+    if (value >= 0) {
+        wheelPos = 0x01 + ((speed * (value * duration)) % 0x48);
```

Comment:
`speed` and `0x48` look like magic numbers. Is there any math behind these numbers, or did you just guess and check until it looked like it worked right?

For the clarity of the code, it would be helpful to have an intermediate variable for `value * direction` called something like `playPositionInSeconds`.